paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -249,3 +254,16 @@ class Paparazzi(
     internal val TIME_OFFSET_NANOS = TimeUnit.HOURS.toNanos(1L)
   }
 }
+
+internal fun detectHandler(mode: Mode) =
+  when (mode) {
+    DEVELOPMENT -> HtmlReportWriter()
+    RECORD -> SnapshotWriter()
+    VERIFY -> SnapshotVerifier()
+  }
+
+enum class Mode {
+  DEVELOPMENT,
+  RECORD,
```

Comment:
Craxy idea: can we combine DEVELOPMENT and RECORD modes? I think we can! All we need to do is emit the HTML files that you view to a different directory from the PNG files you commit